incubator/hnc/pkg/validators/hierarchy_test.go

```diff
@@ -10,16 +10,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
 	api "github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/api/v1alpha1"
-	"github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/pkg/forest"
+	"github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/pkg/foresttest"
 )
 
 func TestStructure(t *testing.T) {
-	f := forest.NewForest()
-	// Create the following forest: foo -> bar
-	foo := createNS(f, "foo", nil)
-	bar := createNS(f, "bar", nil)
-	createNS(f, "baz", nil)
-	bar.SetParent(foo)
+	f := foresttest.Create("-a-") // a -> b; c
```

Comment:
nit: s/a->b/a<-b to match what in the `foresttest.Create` examples (arrow pointing to the parent).

Reply:
Good catch, fixed